DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/RemoveRegistrationPrompt.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+@using Microsoft.Extensions.Configuration
+@model RemoveRegistrationPromptViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Remove delegate registration prompt" : "Remove delegate registration prompt";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var confirmError = ViewData.ModelState[nameof(RemoveRegistrationPromptViewModel.Confirm)]?.Errors?.Count > 0;
+  var confirmFormErrorClass = confirmError ? "nhsuk-form-group--error" : "";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@if (errorHasOccurred) {
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <vc:error-summary order-of-property-names="@new []{ nameof(RemoveRegistrationPromptViewModel.Confirm) }" />
+    </div>
+  </div>
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Remove delegate registration prompt</h1>
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+    <div class="nhsuk-u-font-weight-bold">
+      Prompt:
+    </div>
+  </div>
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l nhsuk-u-font-weight-normal">
+    <input type="hidden" asp-for="PromptName" />
+    @Model.PromptName
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full nhsuk-body-m">
+    Some users have answered this prompt. By deleting this prompt, all the answers that the delegates have given will be deleted too.
+  </div>
+</div>
+
+<form method="post" asp-action="RemoveRegistrationPrompt">
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <div class="nhsuk-form-group @confirmFormErrorClass">
+        <fieldset class="nhsuk-fieldset" aria-describedby="confirm-error">
+          @if (confirmError)

```

Comment:
Do you think it's worth creating a common component for checkboxes too?

Reply:
It's a bit harder than with the other components due to the need to handle multiple checkboxes in a group (and that they would usually not have any errors associated with them). I think we only have the option of either creating a Confirm checkbox component that covers all the stuff here but is not really reusable, or a component for the checkbox item nhsuk-checkboxes__item which feels a bit pointless as it is small.